resources/build/build-utils.sh

```diff
@@ -744,10 +824,19 @@ builder_report() {
     target="$3"
   fi
 
-  if [ $result == success ]; then
-    echo "${COLOR_GREEN}## $action$target completed successfully${COLOR_RESET}"
-  else
-    echo "${COLOR_RED}## $action$target failed. Result: $result${COLOR_RESET}"
+  if [ -n "$_builder_report_scope" ]; then
+    scope="[$THIS_SCRIPT_IDENTIFIER] "
+  fi
+
+  if _builder_item_in_array "$action$target" "${_builder_current_actions[@]}"; then
+    if [ $result == success ]; then
+      echo "${COLOR_GREEN}## $scope$action$target completed successfully${COLOR_RESET}"
+    else
+      echo "${COLOR_RED}## $scope$action$target failed. Result: $result${COLOR_RESET}"
+    fi
+
+    # Remove $action$target from the array; it is no longer a current action
+    _builder_current_actions=( "${_builder_current_actions[@]/$action$target}" )
   fi
```

Comment:
This will cause warnings to be emitted for the current form of `common/web/recorder/build.sh` if given `configure:module` or `configure:proctor` without the other one also being specified.  I intentionally left this out, though something similar _was_ useful when debugging.

Mind if I make that an `elif [[ _builder_debug == true ]];`?